src/scss/_mixins.scss

```diff
@@ -183,15 +183,24 @@
 	}
 
 	.o-teaser--highlight.o-teaser--hero,
+	.o-teaser--special-report-article.o-teaser--hero,
 	.o-teaser--opinion.o-teaser--hero {
 		@include oTeaserHeroThemeBase;
+	}
+
+	.o-teaser--opinion.o-teaser--hero {
 		@include oTeaserHeroOpinion;
 	}
 
-	.o-teaser--highlight.o-teaser--hero {
+	.o-teaser--highlight.o-teaser--hero,
+	.o-teaser--special-report-article.o-teaser--hero {
```

Comment:
Is it important to have this class if it's going to result in the same styles as `.o-teaser--highlight`? 

If it's not going to add anything, would it be possible to just use the highlight class in the presenter?